bedrock/products/templates/products/vpn/landing.html

```diff
@@ -19,14 +19,6 @@
 {% set _utm_campaign = 'vpn-product-page' %}
 {% set _params = '?utm_source=' ~ _utm_source ~ '&utm_medium=referral&utm_campaign=' ~ _utm_campaign %}
 
-{% block experiments %}
-  {% if switch('vpn-landing-page-download-first-experiment') and not switch('vpn-variable-pricing-wave-1') %}
-    {{ js_bundle('vpn-landing-page-download-first-experiment') }}
-  {% elif switch('vpn-landing-page-sub-position-experiment') and not switch('vpn-variable-pricing-wave-1') %}
```

Comment:
Note: I'm not removing the assets for `sub-position-experiment` as we've not yet run the test (but right now we're not ready to because of competing priorities).